test-integration/helpers/task-plan-builder.coffee

```diff
@@ -260,30 +228,60 @@ class ReadingBuilder extends TestHelper
 
     if sections
       # Open the chapter list by clicking the button and waiting for the list to load
-      @openSelectReadingList()
+      if numExercises
+        @openSelectProblems()
+      else
+        @openSelectReadingList()
+
       # Make sure nav bar does not cover buttons
       @test.utils.windowPosition.scrollTop()
 
-      # Expand the chapter and then select the section
-      for section in sections
-        do (section) =>
-          section = "#{section}" # Ensure the section is a string so we can split it
-          @el.selectReadingsList.selectSection(section)
+      @el.selectReadingsList.selectSections(sections)
```

Comment:
I think the conclusion from https://github.com/openstax/tutor-js/pull/983#issuecomment-186302403 (Autogenerating documentation) was that the helpers would not be instantiated and stuck on `@el`. instead it would be something like `(new SelectReadingsList(@test)).selectSections` (or just define the nested helper in PlanBuilder's constructor